postfix/templates/daemonset.yaml

```diff
@@ -47,6 +47,8 @@ spec:
             {{- if .Values.persistence.subPath }}
             subPath: {{ .Values.persistence.subPath }}
             {{- end }}
+          - name: varlog
```

Comment:
If `varlog` only is used with `exporter`, I think it would be better to set  `{{- if .Values.metrics.enabled }}` here too.

Reply:
@cw-sakamoto 
It is not always necessary to log when metrics.enabled is true.
If we're going to make this place dynamic, is there a better way to mount it in `extraVolumes` for those who need it?

Reply:
> It is not always necessary to log when metrics.enabled is true.

OK.

>If we're going to make this place dynamic, is there a better way to mount it in extraVolumes for those who need it?

If `emptyDir` , there's almost no affect, so I think it's fine as is👌

Reply:
@cw-sakamoto 
https://github.com/chatwork/charts/pull/155/commits/436d24e510c93b11e956c90a271996b0e6c29994
It's better to specify your own mount, so I changed it.